--- origsrc/konversation-1.1/konversation/src/Makefile.am	2008-08-06 10:56:27.000000000 -0500
+++ src/konversation-1.1/konversation/src/Makefile.am	2008-08-07 02:11:02.312500000 -0500
@@ -33,7 +33,7 @@
 	connectionsettings.cpp connectionmanager.cpp awaymanager.cpp
 
 konversation_COMPILE_FIRST = config/preferences_base.h
-konversation_LDADD = $(LIB_KIO) $(LIB_XSS) $(LIB_KABC) $(LIB_KIMIFACE) linkaddressbook/liblinkaddressbookui.la blowfish/libblowfish.la config/libkonversationconfig.la
+konversation_LDADD = linkaddressbook/liblinkaddressbookui.la blowfish/libblowfish.la config/libkonversationconfig.la $(LIB_KIO) $(LIB_XSS) $(LIB_KABC) $(LIB_KIMIFACE) 
 
 xdg_apps_DATA = konversation.desktop
 
